### DISCUSSION TITLE
Should the `align` function create a new line?

### DISCUSSION THREAD

**Discussion by lettuceDestroyer** [typst/typst#3982] [General]
I noticed that the `align` function somewhat create a new line. This means when I write something like this:
```typst
#align(left,"Left") #align(right,"Right")
```

The output looks like this:
![image](https://github.com/typst/typst/assets/110594766/3cc5112c-f4a0-4a11-a70c-69e57e99cef3)

What I excepted was something like this:
![image](https://github.com/typst/typst/assets/110594766/901490c7-aee7-4d6d-b9ad-9ea9290dc19e)

What I'm wondering is if that is supposed to be like that or if I can change that somehow.

If anyone is wondering why I would even want to do something like this it's because I wanted to align  data in a table cell without having to use two columns. This would've saved me the overhead of editing the style to fit my needs.

**Comment by PgBiel** [comment]
Use `Left #h(1fr) Right`

**Comment by lettuceDestroyer** [comment]
@PgBiel that is exactly what I needed, thank you 😸
